src/helpers/validators.js

```diff
@@ -1,4 +1,4 @@
-export const signupValidator = (inputObject) => {
+export const signupValidator = inputObject => {
```

Comment:
I realize there the function params does not carry the (). Is this the new es6 syntax for functions

Reply:
Oh yes. For any arrow function that has exactly `one` parameter alone, the parentheses can be omitted. If you check the tests on the server, you'd see that I have been doing that for a while. Also, in the services, you should see some of the functions look that way.